core/src/main/java/org/arquillian/smart/testing/configuration/Report.java

```diff
@@ -0,0 +1,48 @@
+package org.arquillian.smart.testing.configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.arquillian.smart.testing.configuration.Configuration.SMART_TESTING_REPORT_ENABLE;
+import static org.arquillian.smart.testing.report.SmartTestingReportGenerator.REPORT_FILE_NAME;
+import static org.arquillian.smart.testing.report.SmartTestingReportGenerator.TARGET;
+
+public class Report implements ConfigurationSection {
+
+    private boolean enable;
+    private String name = REPORT_FILE_NAME;
+    private String dir = TARGET;
```

Comment:
if the defaults are in `registerConfigurationItems` then they don't have to be here... just a detail...